github/README.md

```diff
@@ -24,3 +24,8 @@ Working on Issues/Pull Requests
 * Submit a Pull Request with the fix.
 * If your Pull Requests fixes one or more issues, link to them in the discussion.
 * Tag a person (or [team](https://github.com/orgs/subvisual/teams)) if you need their input specifically.
+
+Creating a repository
```

Comment:
Beyond the first title I also prefer the `#` syntax, although I have nothing against this one. Yet, you are using different styles here and in `templates/README`, so choose one and stick to it.